cockpit/src/main/res/layout/cockpit_boolean_param.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <include
+        layout="@layout/cockpit_param_name"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="8dp"
```

Comment:
`8dp` could be extracted to `dimens.xml`

Reply:
Layouts are not worth attention. They'll change when Aga provides us final designs.

Reply:
I agree with your second sentence. But it would be much easier to change a single dimension, than replace a lot of `8dp`s in many files.

Reply:
But we don't know if there will be any margin in this place. So we'll be forced to remove it anyways. Constraint layout editor put this 8dp margin by default, so we always need to change it manually.